src/marshmallow/validate.py

```diff
@@ -149,11 +166,17 @@ def __init__(
         self,
         *,
         relative: bool = False,
+        absolute: bool = True,
         schemes: types.StrSequenceOrSet | None = None,
         require_tld: bool = True,
         error: str | None = None,
     ):
+        if not relative and not absolute:
+            raise ValueError(
+                "URL validation cannot set both relative and absolute to False."
+            )
```

Comment:
Should we also prevent `absolute=False` and `require_tld=True`?

Reply:
Yeah, that would make sense. I'll add that shortly. I also realized that I'm missing a test case for this, so I'll be sure to test both of these errors.

Reply:
I went to do this and realized that it won't work cleanly. `require_tld` defaults to `True`, so forbidding the combination makes `URL(absolute=False, relative=True)` fail.

I'd rather allow that usage than catch `absolute=False, require_tld=True` (explicit usage) with an error, so I'm going to leave this change out. But I'll add a test for the existing ValueError.